Gruntfile.js

```diff
@@ -216,7 +216,7 @@ module.exports = function(grunt) {
             s3: grunt.file.readJSON('./cfg/s3.json'),
             timestamp: Date.now(),
             jspmFlags: '-m',
-            assetPath: '<%= visuals.s3.domain %><%= visuals.s3.path %>/<%= visuals.timestamp %>'
+            assetPath: '<%= visuals.timestamp %>'
```

Comment:
As discussed, because of lines 173 and 187 where the assets and the `embed.html` get deployed to different directories, you'd need to go one directory up to successfully use a relative path here